lib/bot.js

```diff
@@ -131,6 +140,30 @@ Bot.prototype.parseText = function(text) {
     });
 };
 
+Bot.prototype.parseMessage = function(message) {
+  return message
+
+    // bold
+    .replace(/(\u0002)([^\u0002]+)(\u0002|\u000f|$)/g, function(match, opener, text, end) {
+      return '*' + text + '*';
+    })
+
+    // italic
+    .replace(/(\u001d)([^\u001d]+)(\u001d|\u000f|$)/g, function(match, opener, text, end) {
+      return '_' + text + '_';
+    })
+
+    // underlined
+    .replace(/(\u001f)([^\u001f]+)(\u001f|\u000f|$)/g, function(match, opener, text, end) {
+      return '_' + text + '_';
+    })
+
+    // remove colour
+    .replace(/(\u0003[0-9]{0,2},?[0-9]{0,2})([^\u0003]+)(\u0003|\u000f|$)/g, function(match, opener, text, end) {
+      return text;
```

Comment:
in our tests the characters showed up. Not the control character itself, but the colour definitions that follow the control character. `\x0301,02foo` showed up as `01,02foo`